lib/moon/design/loader.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Moon.Design.Loader do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  import Moon.Helpers.MergeClass
+
+  prop(size, :string, values: ["2xs", "xs", "md", "lg"], default: "md")
+  prop(color, :string, default: "hit", values: Moon.colors())
+  prop(class, :css_class)
+
+  def render(assigns) do
+    ~F"""
+    <div class={
+      "relative",
+      get_size_class_by_size(@size)
+    }>
+      <div class={loader_class(@class), "border-#{@color}", "border-t-#{@color}", get_border_class_by_size(@size)} />
+      <div class={
```

Comment:
the merge function should be inserted direct here. pretty close to https://github.com/coingaming/moon/blob/main/lib/moon/design/tooltip/arrow.ex#L14
and the class variable should go last, so it can override all previous classes.

Reply:
the same - inside merge